Example/Pods/RichTextVC-iOS/src/Classes/RichTextViewController.swift

```diff
@@ -473,108 +473,127 @@ public class RichTextViewController: UIViewController {
                 }
             }
         }
-
+        
         if range.location != textView.selectedRange.location || range.length != textView.selectedRange.length {
             textView.selectedRange = range
         }
     }
-
+    
     // MARK: Font Adjustments
-
+    
     private func applyFontAttribute(font: UIFont) {
         guard let attributedString = textView.attributedText else { return }
-
+        
         let attributedText = NSMutableAttributedString(attributedString: attributedString)
-
+        
         attributedText.beginEditing()
         attributedText.enumerateAttributesInRange(textView.selectedRange, options: []) { _, range, _ in
             attributedText.addAttribute(NSFontAttributeName, value: font, range: range)
         }
         attributedText.endEditing()
-
+        
         textView.attributedText = attributedText
     }
-
+    
+    private func removeFormattingFromListLeadsInRange(range: NSRange) {
+        guard let listHeadRegex = try? NSRegularExpression(pattern: "^(([0-9]+\\.\\u00A0)|(\\u2022\\u00A0)).*$", options: .AnchorsMatchLines),
+            regularFont = regularFont where
+            range.length > 0
```

Comment:
Its a little weird that this is on the next line, not a deal breaker, but makes it harder to see the `where` condition.